setup.py

```diff
@@ -31,9 +31,8 @@
     keywords=['Kaggle', 'API'],
     entry_points={'console_scripts': ['kaggle = kaggle.cli:main']},
     install_requires=[
-        # Restriction that urllib3's version is less than 1.25 needed to avoid
-        # requests dependency problem.
-        'urllib3 >= 1.21.1, < 1.25',
+        # Restriction that urllib3's version needed to avoid requests dependency problem.
+        'urllib3 >= 1.21.1, < 1.26, != 1.25.0, != 1.25.1'
```

Comment:
There is a comma missing at the end of the line.

Reply:
Thank you for the indication!